core/src/main/scala/com/salesforce/op/stages/sparkwrappers/specific/SparkModelConverter.scala

```diff
@@ -101,4 +100,30 @@ object SparkModelConverter {
     }
   }
 
+  /**
+   * Converts supported mleap loaded spark model of type PredictionModel[Vector, T] to an OP model
```

Comment:
Is this going to be fixed in the next PR?

Reply:
yep